README.md

```diff
@@ -161,9 +167,8 @@ And think If you enjoy gift me a star.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-
 ## 📝 License <a name="license"></a>
 
 This project is licensed under the MIT License - you can click here to have more details [MIT](MIT.md).
```

Comment:
- [ ] The license link is broken:


![image](https://github.com/mohashyne/24hour_doctor-back_end/assets/57421823/f0ef53bc-09cb-4ea3-bebc-92457effd430)

Kindly fix it. 👍